src/geometric_algebra/generic/operators.lean

```diff
@@ -0,0 +1,51 @@
+/-
+Copyright (c) 2020 lean-ga Team. All rights reserved.
+Released under MIT license as described in the file LICENSE.
+Authors: Utensil Song
+
+This file defines operators with local notations and no axioms.
+-/
+import tactic.lint
+
+universes u
+
+namespace geometric_algebra
+
+class has_dot               (α : Type u) := (dot : α → α → α)
+class has_wedge             (α : Type u) := (wedge : α → α → α)
+
+-- class has_geom_prod         (α : Type u) := (geom_prod : α → α → α)
+
+class has_scalar_prod       (α : Type u) := (scalar_prod : α → α → α)
+
+class has_symm_prod         (α : Type u) := (symm_prod : α → α → α)
+class has_asymm_prod        (α : Type u) := (asymm_prod : α → α → α)
+
+class has_left_contract     (α : Type u) := (left_contract : α → α → α)
+class has_right_contract    (α : Type u) := (right_contract : α → α → α)
+
+-- export has_dot (dot)
+-- export has_wedge (wedge)
```

Comment:
This exports a `wedge` function, without `has_wedge`

Reply:
Ah, so I should read this as the python `wedge = has_wedge.wedge`?

Reply:
Not sure about the full extent of this, I saw this in code, check out what it seems to mean and commented it out because I thought it's not useful for our purpose.